MQRS.md

```diff
@@ -73,7 +48,7 @@ And stitching results together with a union.
   * Sometimes an entry gets put into the MQRS database where the DATE_ELIGIBLE is equal to the DATE_CANCELLED. This may cause mis-merges.  You also may want to exclude these completely from your query.
   
   ```
-select * from mqrs.mort_elig_criteria
+select * from NEFSC_GARFO.mqrs_mort_elig_criteria
```

Comment:
I've added this: 
At GARFO, MQRS.MORT_ELIG_CRITERIA is not archived and is a current production table. I strongly suggest using a view to interpret it, because there are some pitfalls in the table structure that aren't obvious.(torey.adler on 15/Dec/23 11:45 AM)

although it's mildly infuriating that all we get are "there are some pitfalls"

Reply:
This is helpful! Thank you. And yes, but I think there are going to be efforts to communicate more thoroughly with GARFO about authoritative data sources where we can address these pitfalls. So, there is that!